rollup.config.js

```diff
@@ -0,0 +1,51 @@
+import path from 'path'
+import svgr from '@svgr/rollup'
+import commonjs from 'rollup-plugin-commonjs'
+import babel from 'rollup-plugin-babel'
+import { terser } from 'rollup-plugin-terser'
+
+import pkg from './package.json'
+
+
+const globals = {
```

Comment:
Globals don't get bundled.  Keeps our size down and prevents possible conflicts with multiple libs getting loaded.